src/jsx/Timeline.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import Post from "./Post.jsx";
+
+class Timeline extends Component {
```

Comment:
watch your spacing in the file.  You've got a mix of tabs/spaces so it throws off formatting depending on the viewer